content/v1.12/concepts/managed-resources.md

```diff
@@ -3,630 +3,745 @@ title: Managed Resources
+  Reason:                ReconcileError
+```
 
-{{<hint "important" >}}
-Management policies are an experimental feature, and the API is
-subject to change.
-{{< /hint >}}
 
-To use management policies, you must enable them with the
-`--enable-management-policies` flag when starting the provider controller.
 
-## Importing Existing Resources
 
-If you have some resources that are already provisioned in the cloud provider,
-you can import them as managed resources and let Crossplane manage them. What
-you need to do is to enter the name of the external resource as well as the
-required fields on the managed resource. For example, let's say I have a GCP
-Network provisioned from GCP console and I would like to migrate it to
-Crossplane. Here is the YAML that I need to create:
+### Success
+`Reason: ReconcileSuccess` indicates the Provider created and is monitoring the 
+managed resource.
 
 ```yaml
-apiVersion: compute.gcp.crossplane.io/v1beta1
-kind: Network
-metadata:
-  name: foo-network
-  annotations:
-    crossplane.io/external-name: existing-network
-spec:
-  forProvider: {}
-  providerConfigRef:
-    name: default
+Conditions:
+  Type:                  Synced
+  Status:                True
+  Reason:                ReconcileSuccess
 ```
 
-Crossplane will check whether a GCP Network called `existing-network` exists,
-and if it does, then the optional fields under `forProvider` will be filled with
-the values that are fetched from the provider.
+### Unavailable
+`Reason: Unavailable` indicates Crossplane expects the managed resource to be 
+available, but the Provider reports the resource is unhealthy.
 
-Note that if a resource has required fields, you must fill those fields or the
-creation of the managed resource will be rejected. So, in those cases, you will
-need to enter the name of the resource as well as the required fields.
+```yaml
+Conditions:
+  Type:                  Ready
+  Status:                False
+  Reason:                Unavailable
+```
 
-### Alternative Import Procedure: Start with ObserveOnly
+### Unknown
+`Reason: Unknown` indicates the Provider has an unexpected error with the
+managed resource. The `conditions.message` provides more information on what
+went wrong. 
+```yaml
+Conditions:
+  Type:                  Unknown
+  Status:                False
+  Reason:                Unknown
+```
 
-Directly importing existing managed resources approach has the following caveats:
 
-1. You must provide all the required fields in the spec of the resource with
-correct values even though they're not used for importing the resource. A wrong
-value for a required field result in a configuration update which isn't
-desired.
-2. Any typos in the external name annotation or mistakes in the identifying
-arguments, such as the `region`, results in the creation of a new resource
-instead of importing the existing one.
+### Upjet Provider conditions
+[Upjet](https://github.com/upbound/upjet), the open source tool to generate
```

Comment:
should we also link to some examples of upjet providers, so readers will know more concretely the type of providers this section applies to?

Reply:
I don't feel strongly but I'm inclined against it. Mainly for the sake of keeping it simple. I'm making an assumption that a user is looking in the docs for their error/condition message as opposed to reading them all cover-to-cover to memorize the conditions.